AirSkyBoat/AirSkyBoat
# Fix Elel Spawn Conditions

**_I affirm:_**


## Please enter a player-facing description


Elel no longer instantly depops on spawn.

## What does this pull request do? (Please be technical)


- Previously forced delay on spawn can cause Elel to pop and instantly depop due to spawn happening outside 20:00-4:00 (Night) Game Time.
- New system pops Elel when conditions are properly met and persists the ToD.
- Added 50 spawn points to nm_spawn_points.sql throughout the maze to be retail accurate (Previously had fixed spawn point and was not listed in nm_spawn_points.sql)


## Steps to test these changes


1 - `!zone Sacrarium`
2 - Wait till 19:00 Game Time
3 - `!setweather GLOOM`
4 - After 20:00 Game Time, `!gotoid 16891948`
Note that Elel will spawn instantly when the correct conditions arrive

Additional Testing 1:
1 - `!zone Sacrarium`
2 - Wait till 3:50 Game Time
3 - `!setweather GLOOM`
4 - `!gotoid 16891948`
Note that Elel will spawn instantly when the correct conditions are already met

Additional Testing 2:
1 - `!zone Sacrarium`
2 - Wait till 4:00 Game Time
3 - `!setweather GLOOM`
4 - `!gotoid 16891948`
Note that Elel will no longer spawn outside the correct conditions (which previously would cause instant depop)

## Special Deployment Considerations



N/A
